cfme/infrastructure/virtual_machines.py

```diff
@@ -10,34 +10,36 @@
 from selenium.common.exceptions import NoSuchElementException
 
 from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic.widget import Text, View
+from widgetastic_patternfly import (
+    Button, BootstrapSelect, BootstrapSwitch, Dropdown, Input as WInput, Tab)
+from widgetastic_manageiq import (
+    TimelinesView, Accordion, ManageIQTree, SummaryTable, ConditionalSwitchableView)
+from widgetastic_manageiq.vm_reconfigure import DisksTable
 
 from cfme.base.login import BaseLoggedInPage
 from cfme.common.vm import VM, Template as BaseTemplate
-from cfme.common.vm_views import ManagementEngineView
-from cfme.exceptions import (CandidateNotFound, VmNotFound, OptionNotAvailable,
-                             DestinationNotFound, TemplateNotFound)
+from cfme.common.vm_views import (
+    ManagementEngineView, ProvisionView, EditView, RetirementView, VMDetailsEntities, VMToolbar,
+    VMEntities)
+from cfme.exceptions import (
+    CandidateNotFound, VmNotFound, OptionNotAvailable, DestinationNotFound, ItemNotFound,
+    VmOrInstanceNotFound)
 from cfme.fixtures import pytest_selenium as sel
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.services.requests import Request
 import cfme.web_ui.toolbar as tb
 from cfme.web_ui import (
-    CheckboxTree, Form, InfoBlock, Region, Quadicon, Tree, accordion, fill, flash, form_buttons,
-    match_location, Table, search, toolbar, Calendar, Select, Input, CheckboxTable,
-    summary_title, BootstrapTreeview, AngularSelect
-)
-from cfme.web_ui.search import search_box
+    CheckboxTree, Form, InfoBlock, Region, Quadicon, Tree, fill, flash, form_buttons,
```

Comment:
:( sad to still see these here, but they need to be removed from common at the same time as here... another day.

Reply:
now they are just in Vm class. I decided not to add it in this PR.